collectors/bzimport/convertors.py

```diff
@@ -750,6 +750,7 @@ def get_meta_attr(self, cve_id):
         meta_attr["original_srtnotes"] = self.flaw_bug["cf_srtnotes"]
         meta_attr["status"] = self.flaw_bug["status"]
         meta_attr["resolution"] = self.flaw_bug["resolution"]
+        meta_attr["fixed_in"] = self.flaw_bug["fixed_in"]
```

Comment:
Make migrations for this to keep the original data.

Reply:
How? This doesn't exist in the database yet. This information is currently thrown out in osidb master.

Reply:
`fixed_in` should be certainly in `flaw.package_versions`. It does not make sense to have `fixed_in` in `meta_attr` if it will be empty for each old flaw (so basically for all flaws now). I would make migrations for all flaws or remove `fixed_in` from `meta_attr`. Otherwise, this would be the only missing part in `meta_attr` for all flaws.

Reply:
> fixed_in should be certainly in flaw.package_versions

This argument would gut `meta_attr` of most of its contents.

> It does not make sense to have fixed_in in meta_attr if it will be empty for each old flaw (so basically for all flaws now).

No. `meta_attr["fixed_in"]` **will not exist** for old flaws. Also please see the explanation in comment https://github.com/RedHatProductSecurity/osidb/pull/307#discussion_r1323373897

> I would make migrations for all flaws or remove fixed_in from meta_attr.

Again, the original `fixed_in` string is currently thrown out in osidb master, so no migration possible. And it must be stored somewhere, because bbsync can't work without it (bbsync already contains a detailed explanation). Again please see the explanation in comment https://github.com/RedHatProductSecurity/osidb/pull/307#discussion_r1323373897

Reply:
> EDIT: If the original fixed_in in BZ contains wrong data (e.g. Upstream: kernel-1.3), then it could be problematic to recreate the original value, that is true. 

Wrong data are out of scope. We can't fix it here.

> In this situation, only resync would make sense for these problematic flaws.

That's still above the manual resync limit specified by Ondrej. Going back to the original issue, how would that be better than having this line in the code?